### ISSUE TITLE
unnamed tests can be misreported as failing

### ISSUE THREAD

**Issue by simonmichael** [joelburget/easytest#7]
It seems that if one unnamed test fails, all unnamed tests in that scope will be reported as failing:
```
tests [
   expect False
  ,expect True
]
->
FAILED
FAILED
```
With names to differentiate them, it works as expected:
```
tests [
   scope "1" $ expect False
  ,scope "2" $ expect True
]
->
FAILED 1
OK 2
```

Perhaps related, would it be helpful for easytest to automatically add numbers like the above to unnamed tests ? I see that it shows "(N)" after OK/FAIL, which is perhaps not quite as useful.

**Comment by joelburget** [comment]
Thanks for the report! I'll be sure to fix this for the next release.

**Comment by alexeyr** [comment]
I just was about to report it and saw this in related issues.

**Comment by joelburget** [comment]
This should be fixed by the 0.3 release.
